package-lock.json

```diff
@@ -1302,6 +1310,14 @@
       "requires": {
         "@gdbots/schemas": "1.6.2",
         "@triniti/schemas": "1.1.3"
+      },
+      "dependencies": {
+        "@gdbots/schemas": {
+          "version": "1.6.2",
```

Comment:
@jalopez13 this seems not good - there is both 1.6.2 and 1.6.3 in here?

Reply:
@efjacobson 

Removed